source/system-administrators/activities/debugging-search-solr.rst

```diff
@@ -97,28 +100,15 @@ Configure Crafter Studio: Crafter Deployer URL
 
 .. code-block:: yaml
 
-  studio:
-    preview:
-      defaultPreviewDeployerUrl: http://HOST:PORT/api/1/target/deploy/preview/{siteName}
-      createTargetUrl: http://HOST:PORT/api/1/target/create
-      deleteTargetUrl: http://HOST:PORT/api/1/target/delete/{siteEnv}/{siteName}
-
---------------------------------------------
-Configure Crafter Engine: Crafter Search URL
---------------------------------------------
-
-``$CATALINA_HOME/shared/classes/crafter/engine/extension/server-config.properties``
-
-.. code-block:: guess
-
-  crafter.engine.search.server.url=http://HOST:PORT/crafter-search
-
-----------------------------------
-Configure Crafter Search: Solr URL
-----------------------------------
-
-``$CATALINA_HOME/shared/classes/crafter/search/extension/server-config.properties``
+  ############################################################
+  ##                    Preview Deployer                    ##
+  ############################################################
+  # Default preview deployer URL (can be overridden per site)
```

Comment:
Now the Deployer URL can be changed in the `crafter-setenv.sh` directly without touching the Studio YAML: https://github.com/craftercms/craftercms/blob/be1833214f07f40913f70c086c22412972f21a1b/resources/env/authoring/bin/crafter-setenv.sh#L49